testutil/verifypr/verify.go

```diff
@@ -168,11 +169,8 @@ func verifyBody(body string) error {
 				ok     bool
 				allows = []string{"feature", "bug", "refactor", "docs", "test", "fixbuild", "misc"}
 			)
-			for _, allow := range allows {
-				if allow == cat {
-					ok = true
-					break
-				}
+			if slices.Contains(allows, cat) {
```

Comment:
```suggestion
			ok = slices.Contains(allows, cat)
```

or combine with the `if` statement below (speaking of greater simplicity)

Reply:
Thanks. Modified.